### ISSUE TITLE
Version 1.1 Candidate: Convert "agent" to "character" in Features - [merged]

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/gsgo#736]
_Merges dev-jd/converting-features-to-character -> dev-next_

Version 1.1 Candidate for Fall 2022 (Oct) Data Collection.

This renames all student-face Feature references to "agent" to "character", but is otherwise identical to `master`.
* `agent` in object references remain `agent` and have not been converted to `character`.  e.g. you would still call `featProp agent.Costume.CostumeName...`

We expect this to be the main branch used during data collection.
Fixes/changes during data collection will be branched off of this branch.

DO NOT merge into `master` until after data collection.
